vision/datasets/cifar10.go

```diff
@@ -0,0 +1,109 @@
+package datasets
+
+import (
+	"fmt"
+	"image"
+	"image/color"
+	"io/ioutil"
+	"os"
+	"path"
+
+	torch "github.com/wangkuiyi/gotorch"
+	"github.com/wangkuiyi/gotorch/vision/transforms"
+)
+
+var cifar10Url = "https://www.cs.toronto.edu/~kriz/cifar-10-binary.tar.gz"
```

Comment:
It would be great if we can have a comment here explaining the format of that tar.gz file.

Reply:
Done, add a link here.